data-access-layer/eventmesh/ApiServerClient.js

```diff
@@ -488,6 +488,75 @@ class ApiServerClient {
       });
   }
 
+  createConfigMapResource(configName, configParam) {
+    logger.info(`Creating ConfigMap ${configName} with data: ${configParam}`);
+    const metadata = {
+      name: configName
+    };
+    let data = {};
+    data = _.set(data, configParam.key, configParam.value);
+    const resourceBody = {
+      apiVersion: CONST.APISERVER.CONFIG_MAP.API_VERSION,
+      kind: CONST.APISERVER.CONFIG_MAP.RESOURCE_KIND,
+      metadata: metadata,
+      data: data
+    };
+    return Promise.try(() => this.init())
```

Comment:
Unncessary `Promise.try` - remove it. init already returns a promise, can just chain it.

Reply:
This is a legacy code, which I got "inspired" from :)
It is present in numerous places in the same file.
As an example: https://github.com/cloudfoundry-incubator/service-fabrik-broker/blob/master/data-access-layer/eventmesh/ApiServerClient.js#L155

Reply:
May be at the locations mentioned its missed earlier reviews. You can fix this instance and lead for a better inspiration  (may be to urself) when you add new methods to this class :-)